rpmconf/Upgrade/zmupgrade.pm

```diff
@@ -289,7 +279,7 @@ sub upgrade {
 
   if (main::isInstalled("zimbra-store")) {
 
-    doMysqlUpgrade() if ($needMysqlUpgrade);
+    doMysqlUpgrade();
```

Comment:
This looks like it might potentially do an expensive operation even if it isn't needed?